### PR TITLE
Enterprise edits for editorial and style

### PR DESCRIPTION
**Pull request by michelle-luna**
Round of editorial changes for style, grammar and headings

### CODE REVIEW

File: jekyll/_ccie/backup.md [hunk 1]
```diff
@@ -1,49 +1,51 @@
...
 section: enterprise
-title: "Backing up CircleCI Enterprise"
+title: "Backing up CircleCI Data"
 category: [advanced-config]
 order: 8
 description: "How to regularly back up CircleCI Enterprise"
 ---
 
-This document will describe how to back up your CircleCI Enterprise application so that you can recover from accidental or unexpected loss of the VM containing all of your persistent CircleCI data (the services box) in the following sections:
+This document describes how to back up your CircleCI Enterprise application so that you can recover from accidental or unexpected loss of the Services machine VM that contains CircleCI data in the following sections:
```
Comment (ndintenfass) [inline jekyll/_ccie/backup.md]: Formally speaking the data lives on a volume attached to the VM. We could probably say something more like "accidental or unexpected loss of CircleCI data attached to the Services machine." (style note: I think "machine" is probably OK by itself - feels bit redundant to say "machine VM"

File: jekyll/_ccie/backup.md [hunk 2]
```diff
@@ -1,49 +1,51 @@
...
 
-The primary, recommended mechanism for backing up your CircleCI data is to use VM snapshots of the virtual disk acting as the root volume for the services box (the machine that serves up the end-user-facing CircleCI UI).
+## Backing up the Database
 
-**Note:** If you are running CircleCI Enterprise in an HA configuration, you must use standard backup mechanisms for the external datastores. See the [High Availability]({{site.baseurl}}/enterprise/high-availability/) document for more information.
+The best practice for backing up your CircleCI data is to use VM snapshots of the virtual disk acting as the root volume for the Services machine. Backups may be performed without downtime as long the underlying virtual disk supports such an operation as is true with AWS EBS. There is a small risk, that varies by filesystem and distribution, that snapshots taken without a reboot may have some data corruption, but this is rare in practice. If zero downtime backups and robustness in the face of data corruption issues are required, then an [HA configuration]({{site.baseurl}}/enterprise/high-availability/) may be the best solution.
```
Comment (ndintenfass) [inline jekyll/_ccie/backup.md]: Consider prefacing "The best practice" with something like "When running CircleCI with the databases on the Services machine, the best practice for..." -- this help set it apart from the "Best practice" of externalizing your databases.

File: jekyll/_ccie/backup.md [hunk 3]
```diff
@@ -1,49 +1,51 @@
...
 
-These backups can be performed without downtime as long the underlying virtual disk supports such an operation (this is true with AWS EBS for example). There is a small risk (that varies by filesystem/distro) that snapshots taken without a reboot may have some data corruption, but this is rare in practice. If zero downtime backups and robustness in the face of data corruption issues are required, then an [HA configuration]({{site.baseurl}}/enterprise/high-availability/) may be the best solution.
+## Backing up Object Storage
 
-## Backing up object storage
+Build artifacts, output, and caches are generally stored in object storage services like AWS S3. These services are considered highly redundant and unnecessary to backup for backup or recovery purposes. An exception is if your instance is setup to store large objects locally on the Services machine, either directly on-disk or on an NFS volume. In this case, you must separately back these files up and ensure they are mounted back to the same location on restore.
```
Comment (ndintenfass) [inline jekyll/_ccie/backup.md]: "backup for backup or recovery..." is a bit awkward - consider "...are considered highly redundant and thus do not likely require separate backup."

File: jekyll/_ccie/backup.md [hunk 4]
```diff
@@ -1,49 +1,51 @@
...
 
-Build artifacts, output, and caches are generally stored in object storage services like AWS S3. These services are generally considered highly redundant and unnecessary to backup for backup/recovery purposes. An exception is if your instance is setup to store large objects locally on the services box, either directly on-disk or on an NFS volume. In this case, you will need to separately back these files up and ensure they're mounted back to the same location on restore.
+## Snapshotting on AWS EBS
```
Comment (ndintenfass) [inline jekyll/_ccie/backup.md]: I am not close enough to the process to comment on the stuff below here -- @bellkev or @GERey should probably weigh in on the details.

File: jekyll/_ccie/proxy.md [hunk 5]
```diff
@@ -7,43 +7,32 @@ order: 7
...
-
-### Service Box configuration
-
-The service box has many components that need to make network calls
-
-- External Network Calls
-  - Replicated
-    - Replicated is a vendor service that we use to bootstrap CircleCI. We require them to make an outside call to validate the license. 
-    - Replicated also goes and downloads docker and installs in on the local machine. They then use a Docker container to create and configure S3 buckets.
+- **External Network Calls** - Replicated is a vendor service that we use for the Management Console of CircleCI. CircleCI  requires Replicated to make an outside call to validate the license. Replicated also downloads docker, installs it on the local machine, and uses a Docker container to create and configure S3 buckets.
```
Comment (ndintenfass) [inline jekyll/_ccie/proxy.md]: after "validate the license" we should add "and check for and download upgrades."

File: jekyll/_ccie/backup.md (hunk 1 above)
Comment (michelle-luna) [inline jekyll/_ccie/backup.md]: thanks, fixed.

File: jekyll/_ccie/backup.md (hunk 2 above)
Comment (michelle-luna) [inline jekyll/_ccie/backup.md]: thanks, I changed to "If you have **not** configured CircleCI for HA, the best practice for backing up your CircleCI data..."

File: jekyll/_ccie/backup.md (hunk 3 above)
Comment (michelle-luna) [inline jekyll/_ccie/backup.md]: thanks, changed to "These services are considered highly redundant and are unlikely to require separate backup."

File: jekyll/_ccie/proxy.md (hunk 5 above)
Comment (michelle-luna) [inline jekyll/_ccie/proxy.md]: added, thanks

**Pull request merged**